backend/src/nodes/nodes/onnx/convert_to_ncnn.py

```diff
@@ -36,8 +40,11 @@ def run(self, model: OnnxModel, is_fp16: int) -> Tuple[NcnnModelWrapper, str]:
         fp16 = bool(is_fp16)
 
         model_proto = onnx.load_model_from_string(model.bytes)
-        passes = onnxoptimizer.get_fuse_and_elimination_passes()
-        opt_model = onnxoptimizer.optimize(model_proto, passes)
+        if onnxoptimizer is not None:
+            passes = onnxoptimizer.get_fuse_and_elimination_passes()
+            opt_model = onnxoptimizer.optimize(model_proto, passes)
```

Comment:
ONNX optimization always seems to be this. Maybe we could move this out into its own shared function to reduce the number of try-import-excepts down to one.